app/tasks/research_tasks.py

```
@@ -517,14 +525,70 @@ def generate_research_report(self, job_id: int) -> Dict[str, Any]:
+        ))
+
+        md_url = b2_result.get("md_url")
+        pdf_url = b2_result.get("pdf_url")
+        md_path = b2_result.get("md_path")
+        pdf_path = b2_result.get("pdf_path")
+
+        # Store report metadata
+        report_record = {
+            "job_id": job_id,
+            "title": f"Research Report: {job['query'][:100]}",
+            "description": f"Generated report for research query",
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)